test/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Tests/Abstractions/IExampleTests.cs

```diff
@@ -34,5 +35,65 @@ public void Given_CamelCaseNamingStrategy_When_Instantiated_Then_It_Should_Retur
             result["first"].Value.Should().BeOfType<OpenApiString>();
             (result["first"].Value as OpenApiString).Value.Should().Contain("{\"number\":");
         }
+
+        [TestMethod]
+        public void Given_IntType_When_Instantiated_Then_It_Should_Return_Result()
+        {
+            var namingStrategy = new DefaultNamingStrategy();
+            var example = new FakeExample();
+
+            var result = example.Build(namingStrategy).Examples;
+
+            result["intValue"].Value.Should().BeOfType<OpenApiInteger>();
+            (result["intValue"].Value as OpenApiInteger).Value.Should().Be(1);
+        }
+
+        [TestMethod]
+        public void Given_StringType_When_Instantiated_Then_It_Should_Return_Result()
+        {
+            var namingStrategy = new DefaultNamingStrategy();
+            var example = new FakeExample();
+
+            var result = example.Build(namingStrategy).Examples;
+
+            result["stringValue"].Value.Should().BeOfType<OpenApiString>();
+            (result["stringValue"].Value as OpenApiString).Value.Should().Be("stringValue");
+        }
+
+        [TestMethod]
+        public void Given_doubleType_When_Instantiated_Then_It_Should_Return_Result()
+        {
+            var namingStrategy = new DefaultNamingStrategy();
+            var example = new FakeExample();
+
+            var result = example.Build(namingStrategy).Examples;
+
+            result["doubleValue"].Value.Should().BeOfType<OpenApiDouble>();
+            (result["doubleValue"].Value as OpenApiDouble).Value.Should().Be(0.123);
+        }
+
+        [TestMethod]
+        public void Given_datetimeType_When_Instantiated_Then_It_Should_Return_Result()
+        {
+            var namingStrategy = new DefaultNamingStrategy();
+            var example = new FakeExample();

```

Comment:
Based on the example class defined above, you should use that to test `DateTime`.